src/utilities/util_dump.c

```diff
@@ -146,12 +165,34 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
         ret = util_err(session, errno, NULL);
 
     __wt_scr_free(session_impl, &tmp);
+    __wt_scr_free(session_impl, &tmp2);
+    __wt_scr_free(session_impl, &timestamp_buf);
     free(uri);
     free(simpleuri);
 
     return (ret);
 }
 
+/*
+ * time_pair_to_timestamp --
+ *     Convert a timestamp output format to timestamp representation.
+ */
+static int
+time_pair_to_timestamp(WT_SESSION_IMPL *session_impl, char *ts_string, WT_ITEM *buf)
+{
+    uint64_t timestamp;
+    uint32_t first, second;
+
+    if (ts_string[0] == '(') {
+        sscanf(ts_string, "(%u,%u)", &first, &second);
```

Comment:
I think you need to pay attention to the return here. According to the man page, you can get 0 due to a matching error.

Reply:
Fixed it.